wfsim/core.py

```diff
@@ -283,7 +283,6 @@ class S1(Pulse):
 
     def __init__(self, config):
         super().__init__(config)
-        # TODO
         # This config is not set for the 1T fax config
         self.config.setdefault('s1_decay_spread', 1)
         self.phase = 'liquid'  # To distinguish singlet/triplet time delay.
```

Comment:
Also for these lines (like the comment below), if we remove the #TODO, can we also remove these two lines?